Paws4Me/Paws4Me/Extensions/UIViewControllerExtension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIAlertExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/09.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController {
+    func displayErrorAlert(alertTitle: String,
```

Comment:
Think about how we can reuse this with enums its almost generic enough to be used for different types of alerts not just errors you are half way there

Reply:
I will definitely look more into it 👍🏻